src/Microsoft.Graph.Core/Requests/Content/BatchRequestContent.cs

```diff
@@ -307,11 +307,10 @@ private string GetHeaderValuesAsString(IEnumerable<string> headerValues)
 
         private string GetRelativeUrl(Uri requestUri)
         {
-            string version = "v1.0";
-            if (requestUri.AbsoluteUri.Contains("/beta/"))//use the url segment to search.
-                version = "beta";
+            if (requestUri == null)
+                throw new ArgumentNullException(nameof(requestUri));
 
-            return requestUri.AbsoluteUri.Substring(requestUri.AbsoluteUri.IndexOf(version, StringComparison.OrdinalIgnoreCase) + version.ToCharArray().Length);
+            return requestUri.PathAndQuery.Substring(5);
```

Comment:
if you're not keeping the comment to explain the magic number, please use a constant with an explicit name

Reply:
Sorry. I just already added the comment in the latest update via https://github.com/microsoftgraph/msgraph-sdk-dotnet-core/pull/312/commits/fd48e1d2b9352779a0d5b30716cea9528fd122bf